### ISSUE TITLE
inputs is empty!

### ISSUE THREAD

**Issue by brian-gyh** [microsoft/nni#5309]
**Describe the bug**:
inputs is empty!As show in figure:
![image](https://user-images.githubusercontent.com/104077097/210755780-0d1adcf4-eaa0-4b27-b683-9d24d5152f5e.png)

node-----> name: .aten::mul.146, type: func, op_type: aten::mul, sub_nodes: ['_aten::mul'], inputs: ['logvar'], outputs: ['809'], aux: None
node-----> name: .aten::exp.147, type: func, op_type: aten::exp, sub_nodes: ['_aten::exp'], inputs: ['809'], outputs: ['std'], aux: None
node-----> name: .aten::randn_like.148, type: func, op_type: aten::randn_like, sub_nodes: ['_aten::randn_like'], inputs: ['mu'], outputs: ['eps'], aux: None
node-----> name: .aten::mul.149, type: func, op_type: aten::mul, sub_nodes: ['_aten::mul'], inputs: ['eps', 'std'], outputs: ['817'], aux: None
node-----> name: .aten::add.150, type: func, op_type: aten::add, sub_nodes: ['_aten::add'], inputs: ['817', 'mu'], outputs: ['819'], aux: None
node-----> name: .aten::unsqueeze.151, type: func, op_type: aten::unsqueeze, sub_nodes: ['_aten::unsqueeze'], inputs: ['819'], outputs: ['821'], aux: None
node-----> name: .aten::unsqueeze.152, type: func, op_type: aten::unsqueeze, sub_nodes: ['_aten::unsqueeze'], inputs: ['821'], outputs: ['z.1'], aux: None
node-----> name: .aten::repeat.153, type: func, op_type: aten::repeat, sub_nodes: ['_aten::repeat', '_prim::ListConstruct'], inputs: ['z.1'], outputs: ['z'], aux: None
node-----> name: .aten::cat.154, type: func, op_type: aten::cat, sub_nodes: ['_aten::cat', '_prim::ListConstruct'], inputs: ['z', 'x_stereo'], outputs: ['input.9'], aux: {'out_shape': [2, 322, 276, 513], 'cat_dim': 1, 'in_order': ['.aten::repeat.153', '.aten::transpose.115'], 'in_shape': [[2, 320, 276, 513], [2, 2, 276, 513]]}
node-----> name: .aten::relu.155, type: func, op_type: aten::relu, sub_nodes: ['_aten::relu'], inputs: ['input.82'], outputs: ['input_tensor'], aux: None
node-----> name: .aten::size.156, type: func, op_type: aten::size, sub_nodes: ['_aten::size'], inputs: ['input_tensor'], outputs: ['1812'], aux: None
node-----> name: .aten::Int.157, type: func, op_type: aten::Int, sub_nodes: ['_aten::Int', '_prim::NumToTensor'], inputs: ['1812'], outputs: ['2374'], aux: None
node-----> name: .aten::size.158, type: func, op_type: aten::size, sub_nodes: ['_aten::size'], inputs: ['input_tensor'], outputs: ['1818'], aux: None
node-----> name: .aten::Int.159, type: func, op_type: aten::Int, sub_nodes: ['_aten::Int', '_prim::NumToTensor'], inputs: ['1818'], outputs: ['2125'], aux: None
node-----> name: .aten::Int.160, type: func, op_type: aten::Int, sub_nodes: ['_aten::Int', '_prim::NumToTensor'], inputs: ['1818'], outputs: ['2119'], aux: None
node-----> name: .aten::size.161, type: func, op_type: aten::size, sub_nodes: ['_aten::size'], inputs: ['input_tensor'], outputs: ['1821'], aux: None
node-----> name: .aten::Int.162, type: func, op_type: aten::Int, sub_nodes: ['_aten::Int', '_prim::NumToTensor'], inputs: ['1821'], outputs: ['2126'], aux: None
node-----> name: .aten::Int.163, type: func, op_type: aten::Int, sub_nodes: ['_aten::Int', '_prim::NumToTensor'], inputs: ['1821'], outputs: ['2120'], aux: None
node-----> name: .aten::slice.164, type: func, op_type: aten::slice, sub_nodes: ['_aten::slice'], inputs: ['input_tensor'], outputs: ['1827'], aux: None

node-----> name: .aten::slice.166, type: func, op_type: aten::slice, sub_nodes: ['_aten::slice'], inputs: ['1827'], outputs: ['1839'], aux: None
node-----> name: .aten::slice.167, type: func, op_type: aten::slice, sub_nodes: ['_aten::slice'], inputs: ['1839'], outputs: ['1844'], aux: None

**Environment**:
- NNI version: 2.10
- Training service (local|remote|pai|aml|etc): remote
- Python version: 3.8.13
- PyTorch version: 1.8.0
- Cpu or cuda version: cuda111

**Reproduce the problem**
- Code|Example: 
According to my position, the mistake should be here:
    def feature_maps_to_wav(
            self,
            input_tensor: torch.Tensor,
            cos_in: torch.tensor,
            sin_in: torch.tensor,
            cos_c: torch.tensor,
            sin_c: torch.tensor,
            audio_length: int,
    ) -> torch.Tensor:
        r"""Convert feature maps to waveform.
        Outputs:
            waveform: (batch_size, output_channels, segment_samples)
         """

        batch_size, _, time_steps, freq_bins = input_tensor.shape

        l_mag = input_tensor[:, [0], :, :]
        r_mag = input_tensor[:, [1], :, :]
        c_mag = input_tensor[:, [2], :, :]
        lfe_mag = input_tensor[:, [3], :, :]
        ls_mag = input_tensor[:, [4], :, :]
        rs_mag = input_tensor[:, [5], :, :]

        lls_cos_in = cos_in[:, 0:1, :, :]
        rrs_cos_in = cos_in[:, 1:2, :, :]
        lls_sin_in = sin_in[:, 0:1, :, :]
        rrs_sin_in = sin_in[:, 1:2, :, :]
        real = torch.cat(
            (l_mag * lls_cos_in,
             r_mag * rrs_cos_in,
             c_mag * cos_c,
             lfe_mag * cos_c,
             ls_mag * lls_cos_in,
             rs_mag * rrs_cos_in),
            dim=1)
        imag = torch.cat(
            (l_mag * lls_sin_in,
             r_mag * rrs_sin_in,
             c_mag * sin_c,
             lfe_mag * sin_c,
             ls_mag * lls_sin_in,
             rs_mag * rrs_sin_in),
            dim=1)

        real = torch.cat((clfe_mag * cos_c, lls_mag * lls_cos_in, rrs_mag * rrs_cos_in), dim=1)
        imag = torch.cat((clfe_mag * sin_c, lls_mag * lls_sin_in, rrs_mag * rrs_sin_in), dim=1)

        real = real.reshape((-1, 1, time_steps, freq_bins))
        imag = imag.reshape((-1, 1, time_steps, freq_bins))

        # ISTFT.
        x = self.istft(real, imag, audio_length)

        # Reshape.
        waveform = x.reshape(batch_size, -1, audio_length)
        return waveform

- How to reproduce: I hope you can help me solve this problem, thanks!

https://github.com/microsoft/nni/issues/5309#tasklist-block-ed4e9d7e-b082-416c-97ab-bcff3aa3c51e
